libraries/botbuilder-dialogs-adaptive/src/actions/emitEvent.ts

```diff
@@ -80,6 +81,11 @@ export class EmitEvent<O extends object = {}> extends Dialog<O> implements EmitE
      */
     public bubbleEvent: BoolExpression;
 
+    /**
+     * The property path to store whether the event was handled or not.
+     */
+    public handledProperty: StringExpression;
```

Comment:
Should this be `public handledProperty?: StringExpression;`?

Reply:
I'm not sure, but that comment did make me realize I forgot to [default it to `turn.eventHandled`](https://github.com/microsoft/botbuilder-dotnet/pull/5446/files#diff-87219dbf070f35ea32d4dd26af07ef5f63f5a8573f7c7249990e01b0e5742e61R97).

If you're questioning the `?` because of:

```js
export interface EmitEventConfiguration extends DialogConfiguration {
    [...]
    handledProperty?: StringProperty;
}
```

...I'm not really sure. The interface and class property optionality is kind of inconsistent across the JS Adaptive library, but it seems like it's _mostly_:

1. Optional in the interface
2. Required in the class property

Reply:
It's likely only required in the class property due to not using `strict` mode. IMO the class and the interface should match. In this case, if there is a default value, the property should _not_ be marked as optional.

Reply:
So, make it required in the `...Configuration` interface, as well, then? None of the class properties are marked optional; should I also make the rest of them required in the interface? Again, normally I would, but there isn't really a pattern in the Adaptive library for this. Actually, I think just about all of the `...Configuration` interfaces across adaptive have all-optional keys.

Reply:
Let's sync up on a call this week to discuss, I think we can make some more general improvements. In the meantime, I won't hold this PR up.